src/app/tables/tests-table/tests-table.component.ts

```diff
@@ -48,6 +48,17 @@ export class TestsTableComponent implements OnChanges {
       .map(waiver => waiver.testcase);
 
     this.formattedResults = this.greenwaveDecision.results.map(result => {
+
+      /* showing only the last result */
+      const sameTestcase = this.greenwaveDecision.results.filter(e => e.testcase.name === result.testcase.name);
+      if (sameTestcase.length > 1) {
+        sameTestcase.sort(
+          (a, b) => (new Date(a.submit_time) > new Date(b.submit_time)) ?
+          -1 : ((new Date(b.submit_time) > new Date(a.submit_time) ? 1 : 0)));
+          if (sameTestcase[0].id !== result.id) {
```

Comment:
Could you unindent this by two spaces? It makes it look like it's part of the sort, but it's not.